scionlab/forms/attachment_conf_form.py

```diff
@@ -261,13 +261,14 @@ class AttachmentConfForm(forms.ModelForm):
 
     class Meta:
         model = Link
-        fields = ('id', 'active')
+        fields = ('id', )
```

Comment:
Is the comma necessary?

Reply:
Yes, that's the normal way to specify a tuple with one element. Otherwise it's parsed as an parenthesized string expression.

Reply:
Ahhh, right. Thanks for the reminder.